src/utils/theme_utils.ts

```diff
@@ -96,20 +93,4 @@ export function blendColors(background: string, foreground: string, opacity: num
     );
 
     return `rgba(${red},${green},${blue},${alpha})`;
-}
-
-// setThemeDefaults will set defaults on the theme for any unset properties.
-export function setThemeDefaults(theme: Theme): Theme {
```

Comment:
0/5 on this right now, but if we leave this function around for the time being, it'll mean that this PR can be merged without breaking the web app. As it is, this PR needs to be merged at the same time as the web app one since anyone else trying to use a new version of mattermost-redux will get an error if the webapp PR hasn't gone in yet.